mmv1/templates/terraform/examples/compute-reservation.tf.erb

```diff
@@ -0,0 +1,21 @@
+# [START compute_reservation_create_local_reservation]
+
+resource "google_compute_reservation" "<%= ctx[:primary_resource_id] %>" {
+  name = "<%= ctx[:vars]['gce_reservation_local'] %>"
+  zone = "us-central1-c"
+  project = "my-project-name"
```

Comment:
I think we'll want to use something like `"<%= ctx[:test_env_vars]['project'] %>"` here, similar to the following:
https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/products/cloudrun/terraform.yaml#L87

this will read the environment variable's project for testing purposes